test/browser-tests/window-utils/serviceworkers/immediate-control.js

```diff
@@ -0,0 +1,2 @@
+self.addEventListener('install', () => self.skipWaiting());
```

Comment:
But we've seen the difference in behavior - so it's not "required" it does affect the end behavior.